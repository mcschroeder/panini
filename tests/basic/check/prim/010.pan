f10 : unit
f10 = unit

f11 : bool
f11 = true

f12 : int
f12 = 1

f13 : string
f13 = "hello"

f14 : char
f14 = 'a'

f15 : string
f15 = "a"
