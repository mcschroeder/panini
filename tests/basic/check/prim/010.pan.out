f10 : 𝟙
f11 : 𝔹
f12 : ℤ
f13 : 𝕊
f14 : ℂ𝕙
f15 : 𝕊